scripts/create-assets.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+function err() {
+  echo >&2 "$(tput setaf 1)ERROR: $*$(tput sgr0)"
+  exit 1
+}
+
+function info() {
+  echo "$(tput bold)$*$(tput sgr0)"
+}
+
+[[ -d tools ]] || err "'tools' directory not found, run script from bpftrace root dir"
+[[ -d man ]] || err "'man' directory not found, run script from bpftrace root dir"
+[[ -f bpftrace ]] || err "bpftrace binary not found, download the build artifact"
```

Comment:
Might be better to figure out repo root from script path (`$0`) so script can be run from another dir.

Reply:
I considered it but was hoping to keep this script minimal in options. I only need this a few times a year, copying the binary/symlinking the binary isn't that much work